### DISCUSSION TITLE
workspace layout

### DISCUSSION THREAD

**Discussion by fedoranvar** [wezterm/wezterm#1719] [Q&A]
Good day to you!

is it possible to define workspace layout (ideally, in external file), similar to `tmuxp`?

Thank you!

**Comment by wez** [comment]
Not today.  There are a few things that are needed to get there:

* https://github.com/wez/wezterm/issues/674
* https://github.com/wez/wezterm/issues/549
* https://github.com/wez/wezterm/issues/1531
* https://github.com/wez/wezterm/issues/225 to support the above
